packages/nextjs/package.json

```
@@ -52,10 +52,10 @@
     "@types/resolve": "1.20.3",
     "@types/webpack": "^4.41.31",
     "eslint-plugin-react": "^7.31.11",
-    "next": "10.1.3"
+    "next": "13.2.0"
   },
   "peerDependencies": {
-    "next": "^10.0.8 || ^11.0 || ^12.0 || ^13.0 || ^14.0",
+    "next": "^13.2.0 || ^14.0",
```

IMO we should make this `14.1.3` for https://github.com/vercel/next.js/pull/61194

```suggestion
    "next": "^13.2.0 || ^14.1.3",
```

---

I don't understand how that would be useful. Can you elaborate? We still have to maintain anything pre 14.1.3.

---

Well if we make `14.1.3` the minimum for `v14`, we don't have to maintain anything pre `14.1.3`. It incentivizes people to use the framework version that works best for the SDK, but I can also see how it introduces friction.